pkg/Microsoft.Private.CoreFx.OOB/Microsoft.Private.CoreFx.OOB.pkgproj

```diff
@@ -11,10 +11,24 @@
       <Version>$(_PreReleasePackageVersion)</Version>
     </PrereleaseLibraryPackage>
   </ItemDefinitionGroup>
-  <ItemGroup>
-    <PkgProjGlobPath Include="$(SourceDir)/**/*.pkgproj" />
-    <PrereleaseLibraryPackage Include="@(PrereleaseLibraryPackage -> %(Filename))" />
-    <Dependency Include="@(PrereleaseLibraryPackage)" />
-  </ItemGroup>
+
+  <Target Name="FindNetCoreAppPackages" BeforeTargets="ApplyBaselineToStaticDependencies">
```

Comment:
Isn't there another more general target that you can hook yourself into, I thought we had something like CalculatePackageDependencies or get PackageDependencies or something like that.

Reply:
GetPackageDependencies